common/app/model/TopMentionsModel.scala

```diff
@@ -3,6 +3,16 @@ package model
 import model.TopMentionEntity.TopMentionEntity
 import play.api.libs.json.{Format, Json}
 
+case class TopMentionFilter(
+    name: String,
+    `type`: TopMentionEntity,
+    count: Int,
+)
+
+object TopMentionFilter {
+  implicit val TopMentionJf: Format[TopMentionFilter] = Json.format[TopMentionFilter]
```

Comment:
is Jf a typo? can't work out what is stands for

Reply:
This follows the existing convention in this file. I believe it stands for Json format